script/jenkins/windows-test.ps1

```diff
@@ -28,7 +28,7 @@ $env:PATH = "$env:GOPATH\bin;C:\tools\mingw64\bin;$env:PATH"
 $env:MAGEFILE_CACHE = "$env:WORKSPACE\.magefile"
 
 # Setup Python.
-exec { choco install python2 -y -r --no-progress --version 2.7.17 }
+exec { choco install python2 -y -r --force --no-progress --version 2.7.17 }
```

Comment:
OTOH, Python 2.7.18 is installed in the workers. We could simply remove the install command and use workers' Python. Thoughts?

Reply:
I guess, we could do so

Reply:
Will wait for the @elastic/apm-server team for clarification about the desired Python version. Will remove the installation  if they agree.

Reply:
that should be fine, but going to merge this nevertheless 👍